common/browser.go

```diff
@@ -297,6 +297,26 @@ func (b *Browser) onAttachedToTarget(ev *target.EventAttachedToTarget) {
 	}
 }
 
+// attachNewPage registers the page as an active page and attaches the sessionID with the targetID.
+func (b *Browser) attachNewPage(p *Page, ev *target.EventAttachedToTarget) {
+	targetPage := ev.TargetInfo
+
+	// Register the page as an active page.
+	b.pagesMu.Lock()
+	b.logger.Debugf("Browser:onAttachedToTarget:addTarget", "sid:%v tid:%v pageType:%s",
```

Comment:
Possibly pretty minor in terms of efficiency, but could we log the debug logs outside of the lock/unlock block?

Reply:
Makes sense 👍  See: 745cf5499